ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -61,3 +82,32 @@ func readECSParams(filename string) (*ECSParams, error) {
 
 	return ecsParams, nil
 }
+
+func ConvertToEcsNetworkConfiguration(ecsParams *ECSParams) *ecs.NetworkConfiguration {
+	networkConfig := ecsParams.RunParams.NetworkConfiguration
```

Comment:
Will this function throw a null pointer exception if Network config or anything it contains doesn't exist?

Reply:
No -- ECSParams is always initialized to its empty values. I added a test for this in ecs_params_reader_test.go

Reply:
👍